doc/usage/extensions/autodoc.rst

```diff
@@ -166,11 +166,19 @@ inserting them into the page source under a suitable :rst:dir:`py:module`,
      This can be combined with ``undoc-members`` to document *all* available
      members of the class or module.
 
+     It can take an anchestor class not to document inherited members from it.
+     By default, members of ``object`` class are not documented.  To show them
+     all, give ``None`` to the option.
```

Comment:
I suggest appending 2 example sentences to avoid misunderstanding by users.

-------

For example; If your class `Foo` is derived from `list` class and you don't want to document `list.__len__`, you should specify a option `:inherited-members: list` to avoid special members of `list` class.
Another example; If your class `Foo` has `__str__` special method and autodoc directive has both `inherited-members` and `special-members`, `__str__` will be documented as in the past, but other special method that are not implemented in your class `Foo`.

Reply:
added.